python/ray/experimental/serve/global_state.py

```diff
@@ -0,0 +1,80 @@
+import time
+from collections import defaultdict, deque
+
+import ray
+from ray.experimental.serve.kv_store_service import KVStoreProxyActor
+from ray.experimental.serve.queues import CentralizedQueuesActor
+from ray.experimental.serve.utils import logger
+from ray.experimental.serve.server import HTTPActor
+
+# TODO(simon): this will be moved in namespaced kv stores
+
+
+class GlobalState:
+    """Encapsulate the global state in the serving system.
+
+    Warning:
+        Currently the state resides inside driver process. The state will be
+        moved into a key value stored service AND a supervisor service.
+    """
+
+    def __init__(self):
+        #: holds all actor handles.
+        self.actor_nursery = []
```

Comment:
I like the name but it doesn't make sense semantically unless these are only actors in startup or something?

Reply:
Newly deployed functions (via `TaskRunner`) and/or actors are added to this nursery. This matters when we are going to add or remove replicas.